### PR TITLE
Update showcase.html

### PR DESCRIPTION
**Pull request by fahminurk**

### CODE REVIEW

**Comment by fahminurk** [comment]
wibupedia

File: showcase.html [hunk 1]
```diff
@@ -128,6 +128,13 @@ <h1>Showcase</h1>
     </header>
 
     <section class="showcase-page">
+               <a class="item" href="https://wibupedia.vercel.app/">
+            <img class="lazy" data-src="https://animeflow.netlify.app/public/preview.png" data-srcset="https://animeflow.netlify.app/public/preview.png 1x" src="" alt="wibupedia" />
+            <div>
+                <h6>wibupedia</h6>
+                <p>Discover your anime favorite</p>
+            </div>
+        </a>
```
Comment (irfan-dahir) [inline showcase.html:137]: Hi, please resolve the following before I can go ahead and merge:

1. Could you please move this entry to the end of the list?
2. The image is returning 404

**Review by irfan-dahir — CHANGES_REQUESTED** [review]

**Comment by fahminurk** [comment]
done

**Comment by fahminurk** [comment]
done

**Pull request merged**